dragonfly/world/chunk/chunk.go

```diff
@@ -5,6 +5,8 @@ import (
 	"sync"
 )
 
+var AirRuntimeID uint32
```

Comment:
I'd rather have this in an internal package.

Reply:
Moved to world internal